e2e/tests/home/home-page.spec.ts

```diff
@@ -98,7 +103,11 @@ describe('Home page ui tests', () => {
   test('Check Workspace card on Home page', async () => {
     await GoogleLoginPage.logIn(page);
 
-    await WorkspaceCard.getAllCards(page);
+    const cards = await WorkspaceCard.getAllCards(page);
```

Comment:
Same critique as above (and applies equally to all of the below early returns): I think it's incorrect to be returning early from a test, and skipping downstream calls to `expect`.